tensorflow/python/kernel_tests/BUILD

```diff
@@ -3216,6 +3216,25 @@ cuda_py_test(
     ],
 )
 
+cuda_py_test(
+    name = "sparse_tensor_dense_matmul_op_test",
+    size = "medium",
+    srcs = ["sparse_tensor_dense_matmul_op_test.py"],
+    tfrt_enabled = True,
+    deps = [
+        ":sparse_tensor_dense_matmul_op_base",
+    ],
+)
+
+cuda_py_test(
+    name = "sparse_tensor_dense_matmul_op_deterministic_test",
+    size = "small",
+    srcs = ["sparse_tensor_dense_matmul_op_deterministic_test.py"],
+    deps = [
+        ":sparse_tensor_dense_matmul_op_base",
+    ],
+)
```

Comment:
If you have not done so already, you should try running this test with XLA auto-jit enabled. I predict that it will fail because XLA does not generate the same d9m-unimplemented exceptions.

Reply:
There's actually a lot to discuss on this point. Please schedule a one-on-one with me.